### ISSUE TITLE
Latex card works in PC but not in phone

### ISSUE THREAD

**Issue by antoniogamiz** [ankidroid/Anki-Android#5454]
###### Reproduction Steps

1. Create the following card from the pc.

~~~
[latex]

Usando la correspondiente definición de esperanza en cada caso:

$Var[X]=E[(X-E[X])^2]$

Por el teorema de Köning:

$Var[x]=E[x^2]-E[x]^2$

- No le afectan las traslaciones.

- Las constantes salen al cuadrado.

[/latex]
~~~

2. Sync it with your phone.
3. View it.

###### Expected Result

Latex should be rendered correctly.

###### Actual Result

The card content is rendered correctly in the PC but not in the phone (an error image appears).

###### Research

[x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid

[x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue

[x] I have searched for similar existing issues here and on the user forum

**Comment by mikehardy** [comment]
From an AnkiDroid perspective, now that we have Mathjax I can't imagine using anything. We use the same mathjax package as Anki Desktop so compatibility should be exact. It's available in the 2.9 beta which seems to be just under a week away - https://docs.ankidroid.org/manual.html#betaTesting and has been through quite a bit of public testing at this point, I don't hesitate to suggest it

**Comment by timrae** [comment]
If your latex images are using svg format, then you need to upgrade to 2.9. If you are using png, then maybe you just haven't synced the media, in which case see here:

https://github.com/ankidroid/Anki-Android/wiki/FAQ#why-doesnt-my-sound-or-image-work-on-ankidroid

In general, LaTeX will most likely not be supported going forward as we have Mathjax now.
